BLSQ/iaso
# IA-2156 form attachments front

Allow to upload, delete, create attachment linked to a form

Related JIRA tickets : IA-2156

## Self proofreading checklist


## Changes

- Split form version detail with two tabs, versions and attachments
- Attachments table with sort and pagination
- delete an attachment
- upload/update attachment

## How to test

Go to a detail form page and click on the attachments tab.
Try to upload several times different files.
Upload a same file a again an check that it has updated the previous one
Try yo delete and download a file.

## Print screen / video


https://github.com/BLSQ/iaso/assets/12494624/a1afa16d-b524-43a6-965c-b8969b2511cb
